app/views/layouts/application.html.erb

```diff
@@ -13,9 +13,9 @@
     <%= yield :meta_tags %>
   </head>
   <body class="govuk-template__body">
-    <script>
+    <%= javascript_tag nonce: true do -%>
```

Comment:
I don't believe `-%>` has any benefit over `%>` in rails now. See https://stackoverflow.com/questions/4407862/minus-sign-at-the-end-of-a-erb-sequence

Reply:
Thanks @reggieb 👍  - have updated as per your suggestion.